src/clib/xtg/libxtg.h

```diff
@@ -480,8 +481,8 @@ surf_get_z_from_xy(double x,
                    int yflip,
                    double rot_deg,
                    double *swig_np_dbl_in_v1,  // *p_map_v
-                   long n_swig_np_dbl_in_v1    // nn
-);
+                   long n_swig_np_dbl_in_v1,   // nn
+                   int option);
```

Comment:
Is this a breaking API change?

Reply:
No the C stuff is not ine the public API (consider they private)